app/src/main/java/com/willowtree/vocable/settings/EditCategoriesViewModel.kt

```diff
@@ -144,6 +144,10 @@ class EditCategoriesViewModel : BaseViewModel() {
         return localizedResourceUtility.getTextFromCategory(updatedCategory)
     }
 
+    fun getUpdatedCategory(category: Category): Category {
+        return overallCategories.first { it.categoryId == category.categoryId }
```

Comment:
Can the `first {  }` method return a null? What happens if it can't find it at all? The method signature says it's non-nullable so I figured there would be compiler errors here.